x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -224,22 +225,22 @@ primaryExpression
     | '(' expression ')'                                                             #parenthesizedExpression
     ;
 
+builtinDateTimeFunction
```

Comment:
Why was the order changed?

Reply:
just to have the cast/convert declarations next to each other.